src/scripts.js

```diff
@@ -53,34 +59,40 @@ let stepGoalComparison = document.querySelector('#step-goal-comparison');
 
 let allUsers;
 let luisa;
+let hydrationUser;
+let ourUser;
 
 const getRandomNumber = (array) => {
     return Math.floor(Math.random() * array.length);
 }
 
```

Comment:
It would be nice to get this random number function to work, if we can's we should remember to remove it when we refactor